src/sssom/util.py

```diff
@@ -504,17 +504,39 @@ def compare_dataframes(df1: pd.DataFrame, df2: pd.DataFrame) -> MappingSetDiff:
     return d
 
 
-def dataframe_to_ptable(df: pd.DataFrame, *, inverse_factor: float = None):
+def add_default_confidence(df: pd.DataFrame, confidence: float = None) -> pd.DataFrame:
+    """Add `confidence` column to DataFrame if absent and initializes to 0.95.
+
+    If `confidence` column already exists, only fill in the None ones by 0.95.
+
+    :param df: DataFrame whose `confidence` column needs to be filled.
+    :return: DataFrame with a complete `confidence` column.
+    """
+    # df[CONFIDENCE] = df.get(CONFIDENCE, confidence)
+    if df.get(CONFIDENCE) is not None:
+        df[CONFIDENCE] = confidence * df[CONFIDENCE]
+    df.loc[df[CONFIDENCE].isnull(), CONFIDENCE] = confidence
```

Comment:
Is this doing the exact same thing as this (which is how I thought of the problem):

```
   if 'confidence' not in df.columns:
        df['confidence'] = confidence
    else:
        df['confidence'] = np.where(
            (df['confidence'].between(0, 1)) & (pd.notnull(df['confidence'])),
            df['confidence'] * confidence,
            confidence
        )
```

Reply:
Yes